dev-libs/libusb/patches/libusb-1.0.23.patchset

```diff
@@ -0,0 +1,59 @@
+From 5aa523289f82d1c7f01dde601356214920646542 Wed Jul 31 00:00:00 2019
+From: Ken Mays <kmays2000@gmail.com>
+Date: Thu, Sep 5 2019 01:15:08 +0000
+Subject: Fix Haiku port
+
+diff --git a/libusb/os/haiku_usb_backend.cpp.older b/libusb/os/haiku_usb_backend.cpp
+index 539a996..16d7528 100644
+--- a/libusb/os/haiku_usb_backend.cpp.older
++++ b/libusb/os/haiku_usb_backend.cpp
+@@ -309,7 +309,7 @@ USBDeviceHandle::SetAltSetting(int inumber, int alt)
+ 	return LIBUSB_SUCCESS;
+ }
+ 
+-
++/*
```

Comment:
And this?

Reply:
As far as I can see the code in upstream is there:
https://github.com/libusb/libusb/blob/master/libusb/os/haiku_usb_backend.cpp